src/ProgressOnderwijsUtils/ProgressOnderwijsUtils.csproj

```diff
@@ -1,8 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <Import Project="..\NugetPackagesCommon.props" />
   <PropertyGroup Label="Configuration">
-    <Version>80.0.1<!--todo bump major version--></Version>
-    <PackageReleaseNotes>Utils.Lazy return Func&lt;&gt; intead of Lazy&lt;&gt;.</PackageReleaseNotes>
+    <Version>81.0.0</Version>
+    <PackageReleaseNotes>Utils.Lazy returns Func&lt;&gt; intead of Lazy&lt;&gt; and no longer caches exceptions.  Utils.Swap removed (can be replaced by tuple assignment).  ToSet removed (can be replaced with Linq ToHashSet).</PackageReleaseNotes>

```

Comment:
Kan aan mij liggen., maar ik zie die genoemde Utils.Lazy wijziging helemaal niet terug in de code

Reply:
losse PR. #614